modules/gitops-release-notes-1-7-0.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+:_content-type: REFERENCE
+
+[id="gitops-release-notes-1-7-0_{context}"]
+= Release notes for {gitops-title} 1.7.0
+
+{gitops-title} 1.7.0 is now available on {product-title} 4.8, 4.9, 4.10 and 4.11.
+
+[id="new-features-1-7-0_{context}"]
+== New features
+
+The current release adds the following improvements:
+
+* With this update, you can add environmental variables to the Notifications controller. link:https://issues.redhat.com/browse/GITOPS-2313[GITOPS-2313]
+
+* With this update, the default `nodeSelector "kubernetes.io/os": "linux"` will be added to all the workloads such that they only schedule on Linux nodes. Also any custom `nodeSelectors` are added to the default and take precedence if the key is same. link:https://issues.redhat.com/browse/GITOPS-2215[GITOPS-2215]
+
+* With this update, you can set custom `nodeSelectors` in the Operator workloads by editing their `GitopsService` custom resource. link:https://issues.redhat.com/browse/GITOPS-2164[GITOPS-2164]
+
+
+* With this update, you can choose the RBAC policy matcher mode. There are currently two options `glob` and `regex`. `Glob` is the default.link:https://issues.redhat.com/browse/GITOPS-1975[GITOPS-1975]
+
+* With this update, you can customize resource behavior through the use of additional subkeys. These subkeys are `resourceHealthChecks`, `resourceIgnoreDifferences` and `resourceActions`. Each of the subkeys maps directly to their own field in the `argocd-cm` so `resourceHealthChecks` will map to `resource.customizations.health`, `resourceIgnoreDifferences` to `resource.customizations.ignoreDifferences`, and `resourceActions` to `resource.customizations.actions`.
```

Comment:
Suggested version:

* With this update, you can customize resource behavior using the following additional subkeys:
+
|===
| *Subkey* | *Key form* | *Mapped field in `argocd-cm`*
| `resourceHealthChecks` | `resource.customizations.health.<group_kind>` | `resource.customizations.health`
| `resourceIgnoreDifferences` | `resource.customizations.ignoreDifferences.<group_kind>` | `resource.customizations.ignoreDifferences`
| `resourceActions` | `resource.customizations.actions.<group_kind>` | `resource.customizations.actions`
|===
+
[NOTE]
====
In future releases, there is a possibility to deprecate the old method of customizing resource behavior by using only `resourceCustomization` and not subkeys.
====